helpers/integration.go

```diff
@@ -216,7 +214,7 @@ func CalculateIntegrationPipelineRunOutcome(adapterClient client.Client, ctx con
 // GetAppStudioTestResultsFromPipelineRunWithChildReferences finds all TaskRuns from childReferences of the PipelineRun
 // that also contain a TEST_OUTPUT result and returns the parsed data
 func GetAppStudioTestResultsFromPipelineRunWithChildReferences(adapterClient client.Client, ctx context.Context, logger logr.Logger, pipelineRun *tektonv1beta1.PipelineRun) ([]*AppStudioTestResult, error) {
```

Comment:
I think it would make sense to log the discovered results either in this function or in the calling `CalculateIntegrationPipelineRunOutcome()` function.

Reply:
I'll think about it, I have some great plans with that function. If we want to keep this log, maybe I'll just close this PR

Reply:
reworked to logs tasks